packages/deployment/app/cloudformation.template.yml

```diff
@@ -394,9 +377,8 @@ Resources:
     Type: AWS::Lambda::EventSourceMapping
     Properties:
       EventSourceArn:
-        Fn::GetAtt:
-          - DataStorageNestedStack
-          - Outputs.{{this}}DynamoDBStreamArn
+        Ref:
```

Comment:
Is it valid to split the `Ref` over two lines?

Reply:
It seems to be. It's all over our templates, including older commits than mine.

Reply:
Can confirm that the ARNs are added to the Lambda as triggers correctly.